test/parallel/test-whatwg-url-from.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
+const { URL } = require('url');
+
+function t(expectedUrl, actualConfig) {
```

Comment:
We generally write assertions via `actual, expected`, not the other way around like you have.

Reply:
@silverwind I know. It reads way faster when expected url comes first. The difference

```js
t({
  protocol: 'https',
  path: ['a'],
  port: 3000,
  host: 'localhost',
  /* ... */
}, 'https://localhost:3000/a')
```

I think it's harder to read.

Reply:
I see. Well at least the `assert.strictEqual` below is in the right order, so probably fine.